src/test/get-req-query-features.js

```diff
@@ -0,0 +1,63 @@
+import { describe } from 'riteway';
+import { getReqQueryFeatures } from '../get-req-query-features';
+import deepFreeze from 'deep-freeze';
+
+describe('getReqQueryFeatures()', async should => {
+  const { assert } = should('');
+  assert({
+    given: 'no arguments',
+    should: 'return an empty array',
+    actual: getReqQueryFeatures(),
+    expected: []
+  });
+  {
+    const req = {
+      query: null
+    };
+    assert({
+      given: 'incorrectly formed req object',
+      should: 'return an empty array',
+      actual: getReqQueryFeatures(deepFreeze(req)),
```

Comment:
More of a preference thing, but I think moving `deepFreeze` out of assert and to the const declaration makes it easier to read the assert details.

```js
const req = deepFreeze({
  query: null
});
```

What do you think? If you agree, please move them all to the const declaration.

Reply:
Ok yeah that makes sense. I'll move them out.